### DISCUSSION TITLE
Design

### DISCUSSION THREAD

**Discussion by Natashka333** [MatthewBozin/Virtual-Office#11] [Ideas]
Hey! This is Natalie (Natashka). 
Here is the link to Figma where you can find the first draft. There is more work that needs to be done. However, I would love to hear your thought, questions, or concerns. 
https://www.figma.com/file/Tf7tSoFsLOOSqWuekbebDR/TechCollab?node-id=0%3A1&t=kVdIqumDAx8VPCtO-1

**Comment by ghost** [comment]
I really like this design :D
